extension/src/cli/dvc/runner.ts

```diff
@@ -164,8 +164,8 @@ export class DvcRunner extends Disposable implements ICli {
         {
           ...baseEvent,
           duration: stopWatch.getElapsedTime(),
-          exitCode,
-          stderr
+          errorOutput: stderr,
```

Comment:
[F] The runner and viewer show processes in the foreground so there is no change for these